diagnostic.rb

```diff
@@ -2,15 +2,42 @@
 # Diagnostic answers to:
 
 # question 2:
+Licensee.create surname: 'Bishop',
+            given_name: 'Caren',
+            gender: 'f',
+            height: 63,
+            weight: 132,
+            born_on: '1943-09-26',
+            eye_color: 'Brown',
+            hair_color: 'Black'
 
 # question 3:
 Licensee.transaction do
   CSV.foreach 'data/licensees.csv', headers: true do |row|
+    person = row.to_hash
+    Person.create! person unless Person.exists? person
   end
 end
 
 # question 4:
+# Use ActiveRecord to retrieve all the licensees with Hazel eye color and
+# Brown or Black hair color.
+Licensee.where( { eye_color: "Hazel", hair_color: ["Brown", "Black"] } )
 
 # question 5:
+# Use ActiveRecord to make each of Marilynn Escobar and Chris Whaley
+# one inch shorter. Look them up by their names.
+marilynn = Licensee.where( { given_name: "Marilynn", surname: "Escobar" } )
+marilynn.update(height -= 1)
+
+chris = Licensee.where( { given_name: "Chris", surname: "Whaley" } )
+chris.update(height -= 1)
```

Comment:
Oh yeah, whoops. Is there a better way to do this that doesn't require me
to do it twice? Like is there a way to use "or" within where, and if so
what's returned by that call? An array?

Thanks!

On Fri, Feb 19, 2016 at 10:50 AM, Matt Brendzel <notifications@github.com>
wrote:

> In diagnostic.rb
> <https://github.com/ga-wdi-boston/rails-activerecord-crud-diagnostic/pull/59#discussion_r53477225>
> :
>
> >
> >  # question 5:
> > +# Use ActiveRecord to make each of Marilynn Escobar and Chris Whaley
> > +# one inch shorter. Look them up by their names.
> > +marilynn = Licensee.where( { given_name: "Marilynn", surname: "Escobar" } )
> > +marilynn.update(height -= 1)
> > +
> > +chris = Licensee.where( { given_name: "Chris", surname: "Whaley" } )
> > +chris.update(height -= 1)
>
> Correct! Just FYI, though, you could also have just written chris.height
> -= 1.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/rails-activerecord-crud-diagnostic/pull/59/files#r53477225>
> .
>